### ISSUE TITLE
testing with elasticface

### ISSUE THREAD

**Issue by dakdouky** [LSIbabnikz/FaceQAN#1]
Hi, 
Thanks for your sharing your work. 

I see from the paper that you use elasticface recognition model, can you share how to use it instead of cosface in the code? 

Thanks!

**Comment by zb4290** [comment]
If you wish to use a different model you have to first obtain the relevant pretrained weights and python script containing the architecture of the model. For ElasticFace you can get this in the following repository https://github.com/fdbtrs/ElasticFace.

Once you have this, you need to simply load the model using PyTorch (using the load_state_dict() function), as is done with CosFace (check out the load_cosface() function in src/models/iresnet.py). Please be aware that not all models use the same normalization, so you might need to change the mean and std values in src/calculate_qs.py line 68.
